CRM/Event/Form/Search.php

```diff
@@ -398,6 +383,42 @@ public function postProcess() {
     $controller->run();
   }
 
+  /**
+   * The post processing of the form gets done here.
+   *
+   * Key things done during post processing are
+   *      - check for reset or next request. if present, skip post procesing.
+   *      - now check if user requested running a saved search, if so, then
+   *        the form values associated with the saved search are used for searching.
+   *      - if user has done a submit with new values the regular post submissing is
+   *        done.
+   * The processing consists of using a Selector / Controller framework for getting the
+   * search results.
+   *
+   * @param
+   *
+   * @return void
+   */
+  public function postProcess() {
+    if ($this->_done) {
+      return;
+    }
+
+    $this->_done = TRUE;
+    $formValues = array();
+
+    if (!empty($_POST)) {
+      $formValues = $this->controller->exportValues($this->_name);
+      CRM_Contact_BAO_Query::processSpecialFormValue($this->_formValues, array('participant_status_id'));
+    }
+
+    if (empty($this->_formValues)) {
```

Comment:
opps we missed this mistake - fix for the regression at https://github.com/civicrm/civicrm-core/pull/14313/files -